### ISSUE TITLE
How to create the model through the code

### ISSUE THREAD

**Issue by apetoman** [google-ar/sceneform-android-sdk#736]
Excuse me, I know through ShapeFactory makexxx can create a shape, we can get from the network model, I want to know is if there by code to create a model, and then to beautify the model on my own

**Comment by tonilsz** [comment]
Yes, it is, you can follow the rules in ShapeFactory, but you will have to set all programmatically: Points, Triangles, UVs, Normals... 

I don't recommend that instread you need simple objects (In my case I use it to create virtual walls) if you want a complex model is better to create it with a 3D tool.
